examples/bri-3/src/bri/workgroup/workflows/api/workflows.controller.spec.ts

```diff
@@ -81,19 +68,23 @@ describe('WorkflowsController', () => {
       ],
     })
       .overrideProvider(WorkflowStorageAgent)
-      .useValue(new MockWorkflowStorageAgent(mapper))
+      .useValue(mockDeep<WorkflowStorageAgent>())
```

Comment:
is there a reason to use these twice?

Reply:
could you please clarify?

Reply:
I believe Manik misread WorkflowStorageAgent and WorkstepStorageAgent as a same thing.